server/routes/projects.js

```diff
@@ -1,14 +1,39 @@
-import dbConnection from "../services/dbConnection.js";
-
-const connection = new dbConnection();
-
 export default app => {
   app.get("/projects", (req, res) => {
-    connection
-      .query("SELECT * FROM Projects")
-      .then(result => {
-        res.send(result);
-      })
-      .then(() => connection.close());
+    res.send("all projects");
+  });
+
+  app.post("/projects", (req, res) => {
+    const {
+      name,
```

Comment:
You will probably want to create a model for projects, but this can be done in a future task.